### ISSUE TITLE
No "Public" option for Taxonomies

### ISSUE THREAD

**Issue by johnpgreen** [WebDevStudios/custom-post-type-ui#387]
While custom posts surface the "public" parameter as an configurable option, custom taxonomies do not have this option.

Use case: I have a taxonomy that I use to sort posts internally but do not use on the front-end.

**Comment by tw2113** [comment]
Hmm. Not sure how I missed that one. Good catch.

If you're comfortable with filters at all, we have this filter available right before the registration of each taxonomy 

```
https://github.com/WebDevStudios/custom-post-type-ui/blob/master/custom-post-type-ui.php#L359
```

This should hold you over until I can get this taken care of.

**Comment by tw2113** [comment]
Fixed in 1.3.0 branch
